BUILD.bazel

```diff
@@ -811,43 +811,63 @@ cc_library(
 )
 
 # TODO(micafan) Replace cc_binary with cc_test for GCS test.
```

Comment:
remove this TODO?

Reply:
I will change this to do, not just remove it. bazel seems support test group, which can run UT as a batch. I think it is useful for developer. Will support this sooner.

Reply:
sounds good